www/api/taggame

```diff
@@ -11,73 +11,50 @@ apiPageHeader("taggame");
 
 <h2>Protocol</h2>
 
-<p>The taggame API is implemented as an http XML web service at
-<tt>https://ifdb.org/taggame?xml</tt>. All parameters are passed as
-POST data with the HTTP request. As typical for HTTP, there's no
-session state or context; the entire transaction is carried out as
-the single request.
-    
-<h2>Parameter names and values</h2>
-
-<p><b>username</b>: the IFDB username for the account under which the
-listing will be created.  Ordinary text parameter; required.
-
-<p><b>password</b>: the user's IFDB password. Ordinary text parameter;
-required.
-
-<p><b>id</b>: the <a href="https://ifdb.org/help-tuid">TUID</a> for
-the game you want to update. Ordinary text parameter; required.
-
-<p><b>t0</b>: the content of the first tag you want to attach to the
-game. For example, <tt>t0=witchcraft</tt> would attach the "witchcraft"
-tag on the game. Ordinary text parameter; optional.
-
-<p><b>t1</b>, <b>t2</b>, <b>t3</b>: attach additional tags in a single
-request using additional numbered parameters. Ordinary text
-parameters; optional.
+<p>The taggame API is implemented as an http JSON web service at
+<tt>https://ifdb.org/taggame</tt>. All parameters are passed as
+JSON-encoded POST data with the HTTP request.
+
+
+<p>Provide your IFDB email address and password using HTTP Basic
+Authentication, using the <code>Authorization: Basic ...</code> header.
+(Optionally, you can provide <code>"username"</code> and <code>"password"</code>
+keys in the JSON request body, but Basic Authentication is preferred.)
+
+<h2>Example request body</h2>
+
+<p>
+<pre>
+{
+    "id": "<i>the <a href="https://ifdb.org/help-tuid">TUID</a> for the game you want to update; required</i>",
+    "tags": [
+        "one tag",
+        "another tag",
+        "another tag",
```

Comment:
"yet another tag" because there shouldn't be duplicates. Also JSON doesn't support commas after the last item.